src/app/file-manager/file-explorer/file-explorer.component.html

```diff
@@ -17,7 +17,11 @@
       <span>{{node.data.size | filesize}}</span>
     </ng-template>
   </p-column>
-  <p-column field="updated" header="Last Modified"></p-column>
+  <p-column field="updated" header="Last Modified">
```

Comment:
If you use `ng-template` then I think the `field=` here no longer has any effect, correct? So it can be removed, along with the `field="size"` above on line 15.

Reply:
You're right, it hasn't. I deleted those already.